.github/workflows/bump-dependencies.yml

```diff
@@ -0,0 +1,174 @@
+            get_upstream_version: GH_REPO=miurahr/aqtinstall gh release view --json tagName --jq .tagName | sed -re 's/^v//'
+            # The following regexps capture both the *nix and the Windows variable syntax (different case, underscore):
+            local_version_regex: (.*AQTINSTALL_?VERSION\s*=\s*"?)([0-9.]*)("?.*)
+
+          - name: Qt6
+            changelog_name: bundled Qt6
+            get_upstream_version: |
+              latest_minor=$(curl -s https://download.qt.io/official_releases/qt/ | grep -oP 'href="\K[0-9.]+(?=/")' | sort --reverse --version-sort | head -n1);
+              curl -s https://download.qt.io/official_releases/qt/"${latest_minor}"/ | grep -oP 'href="\K[0-9.]+(?=/")' | sort --reverse --version-sort | head -n1
+            # The following regex captures both the *nix and the Windows variable syntax (different case, underscore):
+            local_version_regex: (.*QT[0-9_]+VERSION\s*=\s*"?)(6\.[0-9.]+)("?.*)
+
+          - name: choco-jack
+            changelog_name: bundled JACK (Windows-only)
+            get_upstream_version: |
+              curl -s -o /dev/null --location --range 0-5 --write-out '%{url_effective}' https://community.chocolatey.org/api/v2/package/jack/ |
+                grep -oP '.*/jack\.\K.*(?=\.nupkg)'
+            local_version_regex: (.*JackVersion\s*=\s*"?)([0-9.]+)("?.*)
+
+          - name: choco-jom
+            # not Changelog-worthy
+            get_upstream_version: |
+              curl -s -o /dev/null --location --range 0-5 --write-out '%{url_effective}' https://community.chocolatey.org/api/v2/package/jom/ |
+                grep -oP 'jom\.\K.*(?=\.nupkg)'
+            local_version_regex: (.*JomVersion\s*=\s*"?)([0-9.]+)("?.*)
+
+          - name: NSIS
+            changelog_name: Windows Installer base (NSIS)
+            get_upstream_version: |
+              curl -s -o /dev/null --location --range 0-5 --write-out '%{url_effective}' https://sourceforge.net/projects/nsis/files/latest/download |
+                grep -oP '.*/nsis-\K[0-9.]+(?=-setup\.)'
+            # This pattern is a bit special as it has to match twice in a single line.
+            # Therefore, we have to be very careful to avoid consuming too much pattern space.
+            # This is why a positive lookahead is used instead of direct matching:
+            local_version_regex: (.*"nsis-|.*\/NSIS.20.\/|\/nsis-)([0-9.]+)(".*|(?=\/nsis-)|\.zip.*)
+
+          - name: ASIO-SDK
+            changelog_name: ASIO SDK (Windows-only)
+            get_upstream_version: |
+              curl -s -o /dev/null --location --range 0-5 --write-out '%{url_effective}' https://www.steinberg.net/asiosdk |
+                grep -oP '.*asiosdk_\K.*(?=\.zip)'
+            local_version_regex: (.*["\/]asiosdk_)([^"]+?)(".*|\.zip.*)
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ssh-key: ${{ secrets.BUMP_DEPENDENCIES_SSH_DEPLOY_KEY || 'fail-due-to-missing-ssh-key-as-secret' }}
```

Comment:
@ignotus666 how did you handle pushing to the website app without SSH key? I'd prefer that. Maybe it doesn't work well with branch protections, but you don't push to a protected branch, I think.

Reply:
Ah ok. @hoffie I just read your hint on that. Maybe we can still find a way around that (we might want to use a JSON based Matrix anyway soon to allow a dynamically created matrix).

Reply:
> Maybe we can still find a way around that

The officially recommended workaround is using a bot user. I postpone that topic for Jamulus somehow as it would require creating a user, managing/rotating its password, storing its password somewhere for Team access (Github secrets?), finding a way to handle 2FA for the whole team. Also, such users (and their Personal Access Tokens) don't seem to have fine-grained permissions. You can choose the permissions, but not the subjects (i.e. repos), for example.

Yet another option is creating repo-specific Github apps, which sounds unnecessarily complicated to me.

Therefore, this seemed like the most-straightforward solution to allow pushing to .github/workflows/ and to trigger regular PR workflows (Autobuild).